--- lib/pyzor/client.py	2002-09-08 22:37:15.000000000 +0200
+++ /home/calvin/debian/pyzor-0.4.0/lib/pyzor/client.py	2002-12-29 00:23:31.000000000 +0100
@@ -245,6 +245,8 @@
         runner = InfoClientRunner(self.client.info)
 
         for digest in FileDigester(sys.stdin, self.digest_spec):
+            if digest is None:
+                continue
             for server in self.servers:
                 response = runner.run(server, (digest, server))
 
@@ -260,6 +262,8 @@
         runner = CheckClientRunner(self.client.check)
 
         for digest in FileDigester(sys.stdin, self.digest_spec):
+            if digest is None:
+                continue
             for server in self.servers:
                 response = runner.run(server, (digest, server))
                 
