--- scripts/pyzor.ORIG	2003-05-09 03:31:29.000000000 +0200
+++ scripts/pyzor	2003-05-09 03:31:38.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python2.2
 
 import pyzor.client
 pyzor.client.run()
--- scripts/pyzord.ORIG	2003-05-09 03:31:33.000000000 +0200
+++ scripts/pyzord	2003-05-09 03:31:42.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/python2.2
 
 import os
 import os.path
